sync/class.jetpack-sync-actions.php

```diff
@@ -78,6 +78,17 @@ static function add_sender_shutdown() {
 				defined( 'PHPUNIT_JETPACK_TESTSUITE' )
 			)
 		) ) {
+			if ( Jetpack_Sync_Settings::get_setting( 'async_sender' ) && ! Jetpack_Constants::is_true( 'ALTERNATE_WP_CRON' ) ) {
```

Comment:
I was think about this and I think we should limit this to only .com and regular rest requests for now.